db/migrate/20170901061144_add_column_to_wind_turbine_speed_measurement.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToWindTurbineSpeedMeasurement < ActiveRecord::Migration[5.0]
```

Comment:
Speed is the only field of the WindTurbineSeedMeasurement model. Could we use only one model/table called WindTurbineMeasurement and include there all the fields related to the WindTurbine?